git_wrapper/remote.py

```diff
@@ -65,13 +66,13 @@ def fetch(self, remote="origin"):
             raise exceptions.ReferenceNotFoundException(msg)
 
         try:
-            remote.fetch()
+            remote.fetch(prune=prune, tags=tags, branches=branches)
         except git.GitCommandError as ex:
             msg = (f"Could not fetch remote {remote.name} ({remote.url}). "
                    f"Error: {ex}")
             raise exceptions.RemoteException(msg) from ex
 
-    def fetch_all(self):
+    def fetch_all(self, prune=False, tags=False, branches=False):
```

Comment:
The doc string will need to be updated to explain what the new arguments are and how to use them

Reply:
In case you haven't come across it yet, we publish the docs and API references automatically. So the [method documentation for fetch](https://git-wrapper.readthedocs.io/en/latest/git_wrapper.html#git_wrapper.remote.GitRemote.fetch) will show the new arguments once they're added to the doc string.